pages/utilities/number-base-changer.tsx

```diff
@@ -27,14 +28,21 @@ export default function Base64Encoder() {
     []
   );
 
+  const switchValues = () => {
+    const oldFromBase = fromBase;
+    setToBase(oldFromBase);
+    const oldToBase = toBase;
+    setFromBase(oldToBase);
+  };
```

Comment:
After you implement first comment, then we can have something like this.
It's less complex and easier to read:

```
  const switchValues = useCallback(() => {
    setBase((prev) => ({
      from: prev.to,
      to: prev.from,
    }));
  }, []);
```